vite.config.ts

```diff
@@ -0,0 +1,77 @@
+import { babel } from '@rollup/plugin-babel'
+import graphql from '@rollup/plugin-graphql'
+import inject from '@rollup/plugin-inject'
+import reactRefresh from '@vitejs/plugin-react-refresh'
+import path from 'path'
+import { visualizer } from 'rollup-plugin-visualizer'
+import { defineConfig } from 'vite'
+import checker from 'vite-plugin-checker'
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [
+    reactRefresh(),
+    checker({
+      typescript: true,
+      eslint: { files: ['./src'], extensions: ['.js', '.jsx', '.ts', '.tsx'] },
```

Comment:
I'm getting this overlay for eslint warnings: 
![image](https://user-images.githubusercontent.com/51168865/131965620-c60bd0b2-30a1-4860-b565-da0f3ab042a7.png)
![image](https://user-images.githubusercontent.com/51168865/131965736-89495764-7aaf-4222-a4c2-454f1c38d83f.png)
For better developer experience I think we should see this only when an error occurs(not warning)

Reply:
Yeah this is painful. I don't think our current plugin actually lets use configure this but will look into it

Reply:
Do you think just disabling this overlay altogether and having the errors in the console would be okay?

Reply:
Hmm, nope, I'm a little worried that it could be very misleading. I'd rather keep this overlay

Reply:
Just checked how it works, and changed my mind 😅  maybe in the end let's keep it that way!

Reply:
Also, one thing which I don't like in this solution, that warnings will show only on my terminal, not in the browser console.
But I think we could disable this overlay for now and not focus on this. Let's maybe create an issue to bring back this in the future, wdyt?

Reply:
Yeah, I agree it should be in the console as well. But I also think it works well enough for now (you still have warnings in the build output) and I'd love to tackle this at some later point